astropy/units/deprecated.py

```diff
@@ -62,7 +62,8 @@ def enable():
     units only temporarily.
     """
     # Local import to avoid cyclical import
```

Comment:
This should go down with the change in position

Reply:
✅